.npmignore

```diff
@@ -0,0 +1,7 @@
+src
+cmd
+go.mod
+go.sum
+main.go
+zcli-mac
+zcli-linux
```

Comment:
nejde to udelat jako v gitignore whitelistem?

takhle kdyz pridam neco do repa, tak nesmim to tady zapomenout dopsat

pripadne nejde to nahrani pustit nad nejakym adresarem u ktereho budem vedet, ze jsou v nem jen soubory co chceme, nejaky tempNpmUpload?